include/psp2/avconfig.h

```diff
@@ -65,6 +65,16 @@ int sceAVConfigSetSystemVol(int volume);
  *
  */
 int sceAVConfigMuteOn(void);
+  
+/**
+ * Sets the color space mode on runtime.
+ *
+ * @param colorSpaceMode - Color Space Mode that the screen will be set to (0 defualt, 1 high contrast mode).
```

Comment:
Maybe would be cool to have an enum for those constants.

Reply:
Fixed typo.
How do I add the enum?

Reply:
https://github.com/vitasdk/vita-headers/blob/master/include/psp2/audioout.h#L35-L42 Something like this

Reply:
done